install.sh

```diff
@@ -20,6 +20,35 @@ SYMBOLICATOR_CONFIG_YML='symbolicator/config.yml'
 RELAY_CONFIG_YML='relay/config.yml'
 RELAY_CREDENTIALS_JSON='relay/credentials.json'
 SENTRY_EXTRA_REQUIREMENTS='sentry/requirements.txt'
+MINIMIZE_DOWNTIME=
+
+load_options() {
+  while [[ -n "$@" ]]; do
+    case "$1" in
+	    -h | --help) show_help; exit;;
+	    --minimize-downtime) MINIMIZE_DOWNTIME=1;;
+	    --) ;;
+	    *) echo "Unexpected argument: $1. Use --help for usage information."; exit 1;;
```

Comment:
```suggestion
        -h | --help) show_help; exit;;
        --minimize-downtime) MINIMIZE_DOWNTIME=1;;
        --) ;;
        *) echo "Unexpected argument: $1. Use --help for usage information."; exit 1;;
```

Reply:
Let's just use spaces for consistency :)

Reply:
I see the code mixes 2- and 4-space indents - do you think an .editorconfig (in a separate PR, of course) would help?

Reply:
>I see the code mixes 2- and 4-space indents - do you think an .editorconfig (in a separate PR, of course) would help?

Possibly, for vim users I think 😀 I think ultimately we'd need a formatter and a lint rule for this file.

>And if so, what's the preferred style?

I'd go with 2-spaces to be consistent with the `docker-compose.yml` file (although my Python brain wants 4-spaces)

Reply:
@BYK `.editorconfig` is supported in every editor I know of, including VSCode, all IntelliJ-based editors, Atom, Eclipse, Notepad++, Netbeans, SublimeText, and more. You can also set different formats for different file types (or even different file names).

There are also already linters that check all files against the editorconfig configuration, including:

- https://github.com/editorconfig-checker/editorconfig-checker
- https://github.com/jedmao/eclint
- https://gitlab.com/greut/eclint